lib/fluent/supervisor.rb

```diff
@@ -806,6 +811,53 @@ def configure(supervisor: false)
 
       $log.info :supervisor, 'parsing config file is succeeded', path: @config_path
 
+      # merge additional configuration
+      config_type = if File.extname(@config_path) == '.yaml' || File.extname(@config_path) == '.yml'
+                      :yaml
+                    else
+                      nil
+                    end
+      if @system_config.config_include_dir&.empty?
+        $log.warn :supervisor, 'default configuration include directory was disabled'
+      else
```

Comment:
It looks to me that we don't need to warn a user, since the user has explicitly disabled this feature.